dependent_type_theory.md

```diff
@@ -675,8 +696,8 @@ section useful
 end useful
 ```
 
-When the section is closed, the variables go out of scope, and become
-nothing more than a distant memory.
+When the section is closed, the variables go out of scope, and cannot
+be referenced (the variables are private to the section).
```

Comment:
We have "private" definitions, and the use of "private" here may confuse the reader.
I think we should talk about the "scope" of a variable instead.
@Kha What do you think?

Reply:
The sentence already mentions scope, so I think we can drop the parentheses. Perhaps add "any more" instead.

Reply:
Actually how about we keep it clean and simple with "the variables go out of scope."  I think all programmers know what that means.

Reply:
Will mathematicians as well? I don't see any harm in the "cannot be referenced" part.

Reply:
Ok, added "and cannot be referenced any more."